app/templates/view_service.html

```diff
@@ -71,15 +111,16 @@
       {% include "toolkit/page-heading.html" %}
     {% endwith %}
 
-    {% if service_data['frameworkFramework'] == 'g-cloud' %}
-      {%
-        with
-        url = "/{}/services/{}".format(service_data['frameworkFramework'], service_id),
-        text = "View service"
-      %}
-        {% include "toolkit/secondary-action-link.html" %}
-      {% endwith %}
-    {% endif %}
+    {# links #}
```

Comment:
Same with this - is it just a name to make things easier to read?  Or does it do something?